torch/_dynamo/variables/builtin.py

```diff
@@ -354,6 +354,21 @@ def call_function(
         return super().call_function(tx, args, kwargs)
 
     def _call_min_max(self, tx, a, b):
```

Comment:
Can you add a test?

I'd expect you to need to do something like:
```suggestion
    def _call_min_max(self, tx, a, b=None):
```
Otherwise, I don't know how `b` would be set to None